readme.adoc

```diff
@@ -73,6 +73,12 @@ These algorithms help calculate the similarity of nodes:
 * <<algorithms-similarity-euclidean, Euclidean Distance>> (`algo.similarity.euclidean`)
 * <<algorithms-similarity-overlap, Overlap Similarity>> (`algo.similarity.overlap`)
 
+=== Link Prediction
+
+These algorithms can be used as part of Link Prediction solution:
+
+* <<algorithms-linkprediction-adamic-adar, Adamic Adar Similarity>> (`algo.similarity.adamicAdar`)
+
```

Comment:
should the **Resource Allocation algorithm** feature here too?

Reply:
Yes! Also I have the name of the procedure wrong so I've updated that as well